### ISSUE TITLE
Error while submitting payment

### ISSUE THREAD

**Issue by notadamking** [gitcoinco/web#5313]
I encountered an error while submitting payment for an issue, which caused the payment to fail. This happened after submitting the first of two payments. The first payment still went through. How do I go about getting this 0.55 ETH back?

https://www.blockchain.com/eth/tx/0x75fdf2c1936d616613650d519004764ef8031f62dff36b41a2497e380b00a6bc

**Comment by owocki** [comment]
what error did you see?  looks like both tx's went through to me, and that your bounty was posted https://gitcoin.co/issue/notadamking/tensortrade/30/3546

**Comment by notadamking** [comment]
The bounty was posted because I reposted it again with new ETH. 

Here are the two NEW transactions I had to make to post the bounty:
https://www.blockchain.com/eth/tx/0xe6fc9ca18f336a61d487a2c94b97e2bb9f6b24cb72059cd0120e553a2ad17ac4
https://www.blockchain.com/eth/tx/0x4215335bed5b9684e66648b0cb461d91065cab229a2fc36900cdc086b7029028

Here is my meta-mask history:
<img width="359" alt="Screen Shot 2019-10-12 at 7 57 43 AM" src="https://user-images.githubusercontent.com/14098106/66703330-57122f00-ecc6-11e9-81fd-a0bd748e137f.png">

As you can see I accidentally paid the entire 0.55 fee to Gitcoin via the first transaction, then when I realized this I cancelled the second transaction, which caused an "unknown error" (in a red banner) to show across the top of the screen.

**Comment by owocki** [comment]
just sent you a refund.. here is the txid 0x89794c4ddbaba63f1b27110f93656ab76fb7f1d84c8cc0a71634d406bef24043

would still love to know which error you encountered in the UI that left you stranded.

**Comment by notadamking** [comment]
@owocki Thank you. This is my first time using Gitcoin, I apologize for the confusion.

Here is what happened:

1. I entered a price of 0.50 ETH for the bounty.
2. When I reached the payment portion, the total displayed as 0.55 ETH.
3. When MetaMask opened, the payment showed a total of 0.05 ETH.
4. Upon seeing the seemingly incorrect total of 0.05, I adjusted it to 0.55 and pressed complete.
5. A second MetaMask window opened, with a payment total of 0.50 ETH.
6. At this point I realized I had messed up, so I pressed cancel on the payment.
7. A red error notification showed at the top of the screen, saying an unknown error had occurred during the transaction.
8. I refreshed the page to start over. (Not sure why I did this honestly...)
